components/event/EventCard.js

```diff
@@ -42,7 +42,7 @@ export default function EventCard({ manage, event, usernames }) {
 
   const item = (event) => (
     <div
-      className="py-4 border-l-3 mb-4 pl-2 rounded-lg shadow-lg transition duration-350 dark:bg-primary-medium hover:scale-[.99] hover:shadow-sm duration-500 ease-in-out grow"
+      className="py-4 border-l-3 border-t border-secondary-medium dark:border-primary-low mb-4 pl-2 rounded-lg shadow-lg transition duration-350 dark:bg-primary-medium hover:scale-[.99] hover:shadow-sm duration-500 ease-in-out grow"
       style={{
         borderColor: event.color,
```

Comment:
event color is broken, it is currently null for all events.
@eddiejaoude this field is not in the event model.

This is part of the description & screenshots in #7840.
I will look into this issue further.

Reply:
![Screenshot 2023-07-04 094737](https://github.com/EddieHubCommunity/LinkFree/assets/81948346/eb997253-3ffd-47a1-aa7b-78c7cf4b5b31)
@eddiejaoude @dan-mba

Reply:
How does it look on the event tab in the public profile ?

Reply:
It looks good.
![Screenshot 2023-07-05 133845](https://github.com/EddieHubCommunity/LinkFree/assets/81948346/7af499bc-c2c4-4b41-aff1-dbe28f3c53c6)
![Screenshot 2023-07-05 133902](https://github.com/EddieHubCommunity/LinkFree/assets/81948346/941a21dc-38cc-4cf2-8b5d-7b9e932e895e)

Reply:
The color field is not a 'required' field, so can we add this color as default for dark and light mode.
![Screenshot 2023-07-03 095201](https://github.com/EddieHubCommunity/LinkFree/assets/81948346/88f18e33-057d-46a4-a3fd-36ab8fd01998)
![Screenshot 2023-07-03 095219](https://github.com/EddieHubCommunity/LinkFree/assets/81948346/4f790365-0a50-4359-bf76-b46129b2cbe1)

Reply:
I think keep it optional and add a default for light and dark mode

Reply:
yes, thank you